plugins/modules/vyos_logging_global.py

```diff
@@ -0,0 +1,756 @@
+# set system syslog global marker interval '111'
+# set system syslog global preserve-fqdn
+# set system syslog host 172.16.0.1 facility all protocol 'udp'
+# set system syslog host 172.16.0.1 facility local7 level 'all'
+# set system syslog host 172.16.0.1 port '223'
+# set system syslog user paul facility local7 level 'err'
+# set system syslog user vyos facility local7 level 'debug'
+
+- name: Parse the provided configuration
+  vyos.vyos.vyos_logging_global:
+    running_config: "{{ lookup('file', 'parsed_vyos.cfg') }}"
+    state: parsed
+
+# Module Execution Result:
+# ------------------------
+
+# "parsed": {
+#     "console": {
+#         "facilities": [
+#             {
+#                 "facility": "local6"
+#             },
+#             {
+#                 "facility": "local7",
+#                 "severity": "err"
+#             }
+#         ]
+#     },
+#     "files": [
+#         {
+#             "archive": {
+#                 "file_num": 2
+#             },
+#             "facilities": [
+#                 {
+#                     "facility": "local6",
+#                     "severity": "emerg"
+#                 }
+#             ],
+#             "path": "logFile"
+#         }
+#     ],
+#     "global_params": {
+#         "archive": {
+#             "file_num": 2,
+#             "size": 111
+#         },
+#         "facilities": [
+#             {
+#                 "facility": "cron",
+#                 "severity": "debug"
+#             }
+#         ],
+#         "marker_interval": 111,
+#         "preserve_fqdn": true
+#     },
+#     "hosts": [
+#         {
+#             "facilities": [
+#                 {
+#                     "facility": "all",
+#                     "protocol": "udp"
+#                 },
+#                 {
+#                     "facility": "local7",
+#                     "severity": "all"
+#                 }
+#             ],
+#             "hostname": "172.16.0.1",
+#             "port": 223
+#         }
+#     ],
+#     "users": [
+#         {
+#             "facilities": [
+#                 {
+#                     "facility": "local7",
+#                     "severity": "err"
+#                 }
+#             ],
+#             "username": "paul"
+#         },
+#         {
+#             "facilities": [
+#                 {
+#                     "facility": "local7",
+#                     "severity": "debug"
+#                 }
+#             ],
+#             "username": "vyos"
+#         }
+#     ]
+#   }
+# }
+"""
+RETURN = """
+before:
+  description: The configuration prior to the model invocation.
+  returned: always
+  sample: >
+    The configuration returned will always be in the same format
+     of the parameters above.
+  type: dict
+after:
+  description: The resulting configuration model invocation.
+  returned: when changed
+  sample: >
+    The configuration returned will always be in the same format
+     of the parameters above.
+  type: dict
+commands:
```

Comment:
Not all return values are added here. This should be generated by cli_rm_builder.